src/domain/bitcoin/onchain/index.ts

```diff
@@ -32,7 +48,7 @@ export const BaseOnChainTransaction = ({
     }, []),
 })
 
-export const IncomingOnChainTransaction = ({
+export const OutgoingOnChainTransaction = ({
   confirmations,
   rawTx,
```

Comment:
why should we pass rawTx to an `OutgoingOnChainTransaction` is not obvious to me. I would expect a function named `OutgoingOnChainTransaction` to return to me a list of transactions.

Reply:
I didnt change the original implementation, just matched the previous. If you want we can discuss it in code review and updated in the next PR